packages/grid/_modules_/grid/hooks/utils/useResizeContainer.ts

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import { debounce } from '@material-ui/core/utils';
-import { ElementSize } from '../../models';
+import { ElementSize, GridRowsProp } from '../../models';
 import { useLogger } from './useLogger';
 import { useGridSelector } from '../features/core/useGridSelector';
 import { optionsSelector } from './optionsSelector';
 
 const isTestEnvironment = process.env.NODE_ENV === 'test';
 
-export function useResizeContainer(apiRef): (size: ElementSize) => void {
+export function useResizeContainer(apiRef, rowsProp: GridRowsProp): (size: ElementSize) => void {
```

Comment:
Can we grab this from the state, to be compliant with the refactored hooks?

Reply:
the signature of the hooks coming from #1720 is `useThing(apiRef, props)` as I need the change before it gets updated in state